_config.yml

```
@@ -1,96 +1,37 @@
-# Site settings
-title: "The Union of UK Unicyclists"
-email: webmaster@unicycle.org.uk
-description: > # this means to ignore newlines until "baseurl:"
-  The website of the Union of UK Unicyclists
-baseurl: "" # the subpath of your site, e.g. /blog/
-url: "http://www.unicycle.org.uk" # the base hostname & protocol for your site
-twitter_username: UUUnicyclists
-github_username:  unionofukunicyclists
-facebook_page: theunionofukunicyclists
-google_maps_api_key: "AIzaSyBoPKA9DXHMbdvaqMPWxXGX1rUbAJN5JsI"
+# site settings
+title: The Union of UK Unicyclists
+shorttitle: UUU
+subtitle: The Union of UK Unicyclists
+email: contact@unicycling.org.uk
+description: The Union of UK Unicyclists promotes the sport and art of unicycling in the UK. 
+baseurl: "" # the subpath of your site, e.g. /blog
+url: # the base hostname & protocol for your site
+author:
+street_address:
+city:
+state:
+zip_code:
+country: United Kingdom
+phone:
 
-# Build settings
-markdown: kramdown
+# homepage tiles
+tiles-source: pages # accepts "posts" or "pages"
+tiles-count: 8
+
+# social settings
+500px_url:
+facebook_url: https://www.facebook.com/theunionofukunicyclists/
+github_url:
+gitlab_url: 
+googleplus_url: 
+instagram_url: https://instagram.com
+linkedin_url:
+pinterest_url:
+slack_url:
+twitter_url:
```

We have a twitter account.
```suggestion
twitter_url:https://twitter.com/UUUnicyclists
```